CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
-- Upgrade base docker image to fix the detected high security vulnerability. (refer to [issue](https://github.com/wso2/docker-is/issues/277))
+- Fix the vulnerability issues which detected from Docker OS base image.
```

Comment:
Can't we just stay updating the base os image ?

Reply:
resolved the latest commit.